resources/i18n/es_ES/cura.po

```diff
@@ -561,7 +561,7 @@ msgstr "Conectar a Ultimaker Cloud"
 #: /home/ruben/Projects/Cura/plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py:461
 msgctxt "@action"
 msgid "Don't ask me again for this printer."
-msgstr "No volver a preguntarme para esta impresora."
+msgstr "No volver a preguntarme por esta impresora."
```

Comment:
This was actually correct previously, because of the context. The whole sentence (including the context) could be: "Don't ask me again (to connect to Ultimaker Cloud) for this printer" that can be translated in Spanish to "No volver a preguntarme (sobre conectar a Ultimaker Cloud) para esta impresora"